llama_stack/providers/adapters/memory/pinecone/config.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from pydantic import BaseModel
+
+
+class PineconeRequestProviderData(BaseModel):
+    pinecone_api_key: str
+
+
+class PineconeConfig(BaseModel):
+    dimensions: int
```

Comment:
should be singular (`dimension`) and make default value `1024` like you have in the code?

Reply:
Done